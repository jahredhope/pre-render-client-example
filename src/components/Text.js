import styled from 'react-emotion';

export default styled('div')(
  {
    fontSize: '16px',
    fontWeight: '300',
    lineHeight: '24px',
    wordSpacing: '2px'
  },
  ({ heading }) =>
    heading && {
      fontSize: '24px',
      fontWeight: '400',
      lineHeight: '30px',
      marginBottom: '6px',
      wordSpacing: '0px'
    }
);
