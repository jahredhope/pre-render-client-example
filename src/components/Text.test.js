import React from 'react';
import Text from './Text';
import { shallow } from 'enzyme';

describe('Text', () => {
  it('should render body text', () => {
    expect(shallow(<Text>foo</Text>)).toMatchSnapshot();
  });
  it('should render heading', () => {
    expect(shallow(<Text heading>foo</Text>)).toMatchSnapshot();
  });
});
