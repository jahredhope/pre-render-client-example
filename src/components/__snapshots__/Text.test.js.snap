// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text should render body text 1`] = `
<div
  className="css-1sbhzye ekj1ype0"
>
  foo
</div>
`;

exports[`Text should render heading 1`] = `
<div
  className="css-wcfrey ekj1ype0"
>
  foo
</div>
`;
